### PR TITLE
[Feature] Add Balanced MSE Loss and Imbalanced HMR Task 

### PR DESCRIPTION
**Pull request by mingyuan-zhang**
Reproduction of [Balanced MSE for Imbalanced Visual Regression](https://arxiv.org/abs/2203.16427)
| Method |  bMPVPE | bMPJPE | bPA-MPJPE |
|:------:|:-------:|:------:|:------:|
| baseline | 115.33/126.98/132.20 | 98.69/111.79/113.61 | 65.98/76.89/76.76 |
| bmc | **112.67/126.13/130.42** | **95.70/110.22/111.62** | **65.65/75.61/75.86** |

Modification Log:

1. Add Imbalanced HMR metric
2. Add Balanced MSE Loss (batch mento carlo version)
3. Add configs for ihmr baseline and bmc method
4. Add `keypoint_weight` for `KeypointMSELoss`

Coverage Issue

Metrics for Balanced MSE needs extra data `smpl_mean_params.npz`, whose path is hard code now. During uni-test, I can not assign new path for evaluation due to the lack of configruability of evaluation metrics. Therefore, balanced metrics are not covered by unitest. A new [issue](https://github.com/open-mmlab/mmhuman3d/issues/234) is opened for this problem.

### CODE REVIEW

File: configs/balanced_mse/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# GTA-Human
```
Comment (caizhongang) [inline configs/balanced_mse/README.md:1]: `GTA-Human` -> `Balanced MSE`

**Review by caizhongang — COMMENTED** [review]
After adding the links to files, the PR is ready!

File: configs/balanced_mse/README.md (hunk 1 above)
Comment (mingyuan-zhang) [reply configs/balanced_mse/README.md]: solved

**Comment by mingyuan-zhang** [comment]
> The PR is almost ready! Please do the following:
> 
> 1. Adding the links to files
> 2. Updating the main README / README_CN
> 3. Link an issue about coverage in the description


1.  2. Solved in newest commit
3. Description is updated

**Review by caizhongang — APPROVED** [review]

**Pull request merged**